src/js/DropContainer.jsx

```diff
@@ -0,0 +1,281 @@
+/*eslint consistent-this: [2, "that"]*/
+
+import React from 'react';
+import classNames from 'classnames';
+import accepts from 'attr-accept';
+import $ from 'jquery';
+
+import css from '../stylesheets/button.css';
```

Comment:
Let's rename this to something other than `button.css` as it might be confusing as to what it's pointing at